app/controllers/v0/caregivers_assistance_claims_controller.rb

```diff
@@ -39,19 +39,23 @@ def download_pdf
       # Brakeman will raise a warning if we use a claim's method or attribute in the source file name.
       # Use an arbitrary uuid for the source file name and don't use the return value of claim#to_pdf
       # as the source_file_path (to prevent changes in the the filename creating a vunerability in the future).
-      source_file_path = PdfFill::Filler.fill_form(@claim, SecureRandom.uuid, sign: false)
+      source_file_path = if Flipper.enabled?(:caregiver1010)
```

Comment:
No request specs need updated since the functionality should be exactly the same and we already have some for when the flipper toggle is enabled vs disabled.

Reply:
I added back the request specs with the toggle off/on since they were removed in a separate PR. They are needed again here since we are using the toggle for this logic.